Mlem/App/Views/Shared/Navigation/NavigationPage.swift

```diff
@@ -59,7 +64,10 @@ extension NavigationPage {
         case .inbox:
             InboxView()
         case .search:
-            EmptyView()
+            VStack {
+                Link("Test", destination: URL(string: "https://lemmy.dbzer0.com")!)
+                Text("\((AppState.main.firstSession as? UserSession)?.blocks?.instanceCount)")
```

Comment:
Fails build on Xcode 15

```suggestion
                Text("\((AppState.main.firstSession as? UserSession)?.blocks?.instanceCount ?? 0)")
```